data/campaigns/Eastern_Invasion/scenarios/12_Evacuation.cfg

```diff
@@ -172,185 +268,625 @@
+        [/message]
+        {REPLACE_SCENARIO_MUSIC cry_from_elensefar.ogg}
+        {APPEND_MUSIC legends_of_the_north.ogg}
+        {APPEND_MUSIC frantic.ogg}
+
+        [store_gold] # store this now in case we spend it before the IF_GOLD event
+            side=1
+        [/store_gold]
+    [/event]
+    [event]
+        name=recruit,recall,moveto,side 1 turn end # so it triggers very quickly
+        [if] {VARIABLE_CONDITIONAL gold less_than_equal_to 0}
+            [then]
+                [message]
+                    speaker=Dacyn
+                    message= _ "It is unfortunate (cough) that we have not saved any gold to recruit or recall. Escape for our soldiers is impossible, yet if us few rush immediately across the bridge we may be able to (cough)(cough) reach the far shore before the orcs can attack."
+                [/message]
+            [/then]
+
+            [else]
+                [if] {VARIABLE_CONDITIONAL gold less_than 100}
+                    [then]
+                        [message]
+                            speaker=Dacyn
+                            message= _ "It is unfortunate (cough) that we have saved little gold to recruit or recall. Escape for everyone is likely impossible, yet if the 3 of us rush immediately across the bridge we may be able to (cough)(cough) reach the far shore before the orcs can attack."
+                        [/message]
+                    [/then]
+
+                    [else]
+                        [if] {VARIABLE_CONDITIONAL gold less_than 200}
+                            [then]
+                                [message]
+                                    speaker=Dacyn
+                                    message= _ "We have enough gold to put up a fight, but the orcs are numerous (cough) and strong. Escape for everyone is likely impossible, yet if the 3 of us rush immediately across the bridge we may be able to (cough)(cough) reach the far shore before the orcs can attack."
+                                [/message]
+                            [/then]
+
+                            [else]
+                                [message]
+                                    speaker=Dacyn
+                                    message= _ "We have gold for many men, but the bridge (cough) is long and narrow. Escape for everyone is likely impossible, yet if the 3 of us rush immediately across the bridge we may be able to (cough)(cough) reach the far shore before the orcs can attack."
+                                [/message]
+                            [/else]
+                        [/if]
+                    [/else]
+                [/if]
+            [/else]
+        [/if]
+        [message]
+            speaker=Owaec
+            message= _ "Small wonder your light has abandoned you, if you would leave our companions to the mercy of the orcs! We should all cross together and leave none behind!"
```

Comment:
Dacyn uses 'light' in the generic literal sense after this line, but I think it makes sense to capitalise 'light' here.